lightpath/ui/widgets.py

```diff
@@ -233,6 +234,7 @@ def __init__(self, device, parent=None):
         # Default UI settings for conformity
         self.setMinimumSize(10, 10)
         self.setMaximumSize(50, 50)
+        self.setStyleSheet('padding : 0px')
```

Comment:
Totally optional tangent: 
I had started putting stylesheet stuff in a separate file in btms-ui. 
I think it worked reasonably well:
* https://github.com/pcdshub/btms-ui/blob/master/btms_ui/stylesheet.qss
* https://github.com/pcdshub/btms-ui/blob/c6c6d7aaf95fc39bcd2ac754e1cd01c18af2d82b/btms_ui/main.py#L21

You might consider something similar down the line. It's nice to have all style stuff in one place.